components/Slider/src/MDCSlider.m

```diff
@@ -209,6 +213,25 @@ - (UIColor *)backgroundTrackTickColorForState:(UIControlState)state {
   return color;
 }
 
+- (void)setThumbRadius:(CGFloat)thumbRadius forState:(UIControlState)state {
+  NSAssert(thumbRadius >= 0, @"Thumb radius must be a positive value");
+  float radius = (float)thumbRadius;
```

Comment:
Let's keep this as a CGFloat or change the API to be `float` type. I'd prefer that we retain the full value provided by clients.

Reply:
Done